test/tools/yulopti.cpp

```diff
@@ -167,9 +194,10 @@ class YulOpti
 			}
 			map<char, string> const& abbreviationMap = OptimiserSuite::stepAbbreviationToNameMap();
 			map<char, string> const& extraOptions = {
-				{'#', "quit"},
-				{',', "VarNameCleaner"},
-				{';', "StackCompressor"}
+				// QUIT starts with a non-letter character on purpose to get it to show up on top of the list
```

Comment:
What about adding it manually after sorting?

Reply:
See my comment above. I think `QUIT` should  be either first or last and with what you propose I think I'd have to handle it as a special case in `printUsageBanner()`. By sorting it all together I managed to get away with just changing its label slightly :)